extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeGroup.php

```diff
@@ -42,13 +58,21 @@ public function getSource() {
 		return $this->getSourceForChildren();
 	}
 
-	/**
-	 * @return string
-	 */
+	protected function showIncomplete() {
+		return strcmp( $this->getDisplay(), self::SHOW_INCOMPLETE_OPTION ) === 0;
```

Comment:
An interesting approach, I may be missing why we use strcmp an not a direct string comparison through === - what's the rationale as we're interested only in (in)equality and not its direction?